sdk/remoterendering/Azure.MixedReality.RemoteRendering/src/RemoteRenderingClient.cs

```diff
@@ -204,7 +206,9 @@ Page<AssetConversion> NextPageFunc(string nextLink, int? pageSizeHint)
 
         /// <summary> Gets a list of all conversions.</summary>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
+#pragma warning disable AZC0004 // DO provide both asynchronous and synchronous variants for all service methods.
         public virtual AsyncPageable<AssetConversion> GetConversionsAync(CancellationToken cancellationToken = default)
+#pragma warning restore AZC0004 // DO provide both asynchronous and synchronous variants for all service methods.
         {
```

Comment:
There is a typo here `GetConversionsAync` should be `GetConversionsAsync`. Therefore these two async/sync pair cannot find each other. I have to suppress here. This library is GAed.

Reply:
Thank you for finding this!  It might be helpful to file an issue for this library to let them know - they could introduce a correctly-spelled method and mark the misspelled one with "EditorBrowsableState.Never"